plotly/Cargo.toml

```
@@ -24,7 +24,7 @@ askama = { version = ">=0.11.0, <0.13.0", features = ["serde-json"] }
 dyn-clone = "1"
 erased-serde = "0.3"
 getrandom = { version = "0.2", features = ["js"], optional = true }
-image = { version = "0.24.2", optional = true }
+image = { version = "0.25.1", optional = true }
```

```suggestion
image = { version = ">=0.24, <0.26", optional = true }
```
Depend on versions 0.24 or 0.25, as either works.

---

Unfortunately, version 0.25.x doesn't work out of the box. https://github.com/plotly/plotly.rs/actions/runs/8901657567/job/24445956493

Will have to close this PR only with allowing version 0.24 for now